plugins/org.jkiss.dbeaver.ext.mssql/src/org/jkiss/dbeaver/ext/mssql/model/SQLServerDataSource.java

```diff
@@ -221,6 +220,8 @@ protected Properties getAllConnectionProperties(@NotNull DBRProgressMonitor moni
         final DBWHandlerConfiguration sslConfig = getContainer().getActualConnectionConfiguration().getHandler(SQLServerConstants.HANDLER_SSL);
         if (sslConfig != null && sslConfig.isEnabled()) {
             initSSL(monitor, properties, sslConfig);
```

Comment:
Umm??
CommonUtils.isEmpty(PROP_ENCRYPT_SSL is always false.
Please fix and test it

Reply:
Fixed